pkgs/development/libraries/gvfs/default.nix

```
@@ -120,6 +120,9 @@ stdenv.mkDerivation rec {
     updateScript = gnome3.updateScript {
       packageName = pname;
     };
+    propagateEnv = {
+      GIO_EXTRA_MODULES = "%out%/lib/gio/modules"
```

```suggestion
      GIO_EXTRA_MODULES = "%out%/lib/gio/modules";
```

Minor mistake causing ofborg to cry ;)